docs/release-notes/index.rst

```diff
@@ -0,0 +1,253 @@
+.. _release-notes:
+
+Release notes
+=============
+
+This page lists past release notes for Starcraft, summarising new features, bug
+fixes and backwards-incompatible changes in each version. It also contains the
+release and support policies for Starcraft.
+
+
+Current releases
+----------------
+
+
+<latest release>
+~~~~~~~~~~~~~~~~
+
+- <link to latest release, update, such as 8.2.0>
+- <link to latest release, update, such as 8.1.0>
+- <link to latest release, initial, such as 8.0.0>
+
+
+<parallel release>
+~~~~~~~~~~~~~~~~~~
+
+<If necessary, add guidance and caveats about these older releases, such as
+"Snapcraft 7 is available for building core18 snaps. When building for a newer
+base, use Snapcraft 8.">
+
+- <link to parallel release, update, such as 7.2.0>
+- <link to parallel release, update, such as 7.1.0>
+- <link to parallel release, initial, such as 7.0.0>
+
+
+Past releases
+-------------
+
+
+<past release>
+~~~~~~~~~~~~~~
+
+- <link to past release, update, such as 6.2.0>
+- <link to past release, update, such as 6.1.0>
+- <link to past release, initial, such as 6.0.0>
+
+
+.. _release_policy_and_schedule:
+
+Release policy and schedule
+---------------------------
+
+Canonical is committed to supporting the <"latest major release" or "last two
+major releases"> of Starcraft. <Optional: "We forward-port changes in older
+releases to the latest release, if they're compatible.">
+
+Starcraft is released according to the Semantic Versioning 2.0.0 scheme with
+numbers for major, minor, and patch versions.
+
+.. list-table::
+    :header-rows: 1
+
+    * - Version
+      - Example
+      - Significance
+    * - Major
+      - **3**.1.2
+      - A change that breaks compatibility with the previous version.
+    * - Minor
+      - 3.\ **1**\ .2
+      - A new feature within the major version.
+    * - Patch
+      - 3.1.\ **2**
+      - A bug fix within the major or minor version.
+
+Starcraft is released when it achieves development milestones in its product
```

Comment:
I believe we are also updating this - for instance, starting in 2025 Rockcraft will have [monthly releases](https://discourse.ubuntu.com/t/rockcraft-releases/50007)

Reply:
Yes! That part would need to be customised for Rockcraft's instance of this doc. Will other products follow suite?

Reply:
Maybe! I guess we'll see how it plays out with Rockcraft. I do know that there's a desire to have a more regular cadence for application releases.